### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by Esruzn07**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,9 +3,65 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Stack Over Flow</title>
```
Comment (annasvst) [inline index.html:6]:
```suggestion
    <title>Stack Over Flow</title>
    <link rel="stylesheet" href="./styles.css">

```

File: index.html [hunk 2]
```diff
@@ -3,9 +3,65 @@
...
+    
+            <p>Collaborate and share knowledge with a private group for FREE.</p>
+    
+            <a href="#">Get Stack Overflow for Teams free for up to 50 users.</a>
+          
+        </div>
+    
+      <div class="content--right">
+    
+        <div class="button-signup">
```
Comment (annasvst) [inline index.html:30]:
```suggestion
        <div class="buttons-container">
```

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    max-width: 316px;
+  
+  }
+  
+  ul li { 
+    margin-bottom: 10px; 
+  }
+  
+  
+  .button-signup {
```
Comment (annasvst) [inline styles.css]:
```suggestion
  .buttons-container {
```

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+   display: flex;
+   flex-direction: column;
+   flex-wrap: nowrap;
+   justify-content: space-around;
+   align-items: stretch;
+   align-content: stretch;
```
Comment (annasvst) [inline styles.css]:
```suggestion
```

File: index.html [hunk 5]
```diff
@@ -3,9 +3,65 @@
...
+    
+        <div class="google"><a href="https://myaccount.google.com/"><img src="logo-google.svg" alt="Google Logo" width="20" height="20">Sign up with Google</a></div>
```
Comment (annasvst) [inline index.html:32]: There's no need to wrap <a> elements inside divs. 
```suggestion
       						<a class="btn google" href="https://myaccount.google.com/"
							><img
								src="logo-google.svg"
								alt="Google Logo"
								width="20"
								height="20"
							/>Sign up with Google</a
						>
```

Apply to all 3 buttons here

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+  }
+  
+  
+  .google {
```
Comment (annasvst) [inline styles.css:57]: There's a lot of repetition in google, facebook and github classes. You can move the same code to a generic class, e.g. 
`
  .btn {
    border-radius: 5px;
    margin-bottom: 5px;
    text-align: center;
    text-decoration: none;
    padding: 10px;
  }
`

and then remove these lines from the mentioned classes

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+    background-color: #f1f2f3;
+  }
+  
+  main {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    align-items: center;
+    align-content: stretch;
```
Comment (annasvst) [inline styles.css:31]: Add some space at the top of the page
```suggestion
    align-content: stretch;
    padding-top: 50px;
```

**Review by annasvst — CHANGES_REQUESTED** [review]

File: styles.css (hunk 6 above)
Comment (Esruzn07) [reply styles.css:57]: Hi Anna, thank you I get it but Google, Facebook, Github backgrounds are different color, how can I fix that if I remove it?

Comment (Esruzn07) [reply styles.css:57]: And where should I add .btn class? Is it <a>

Comment (annasvst) [reply styles.css:57]: You should add 'btn' class that will has styles that all buttons have. Then, you add 'btn' class to all buttons that you have in the page.
After, you can add a second class that only has specific styles (e.g. background color).

HTML will look something like this:
`
<a href="#" class="btn google">Sign up with Google</a>`

**Pull request merged**